### PR TITLE
Adds zipkin.web.query.limit flag

### LINKED ISSUE #462: web DefaultQueryLimit is not overridable
`Constants.DefaultQueryLimit` is a hard-coded val. Seems at least this should be overridable by a flag, right?

https://github.com/openzipkin/zipkin/blob/master/zipkin-web/src/main/scala/com/twitter/zipkin/web/Constants.scala#L19

see #443

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Adds `zipkin.web.query.limit` flag to zipkin-web, which allows users
to override the default value of 10.

closes #462

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
Note, I tried to follow existing convention.. this doesn't mean I love it :)

cc @noslowerdna @mosesn @rtyler

File: zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala [hunk 1]
```diff
@@ -55,12 +55,15 @@ trait ZipkinWebFactory { self: App =>
...
   val queryDest = flag("zipkin.web.query.dest", "127.0.0.1:9411", "Location of the query server")
+  val queryLimit = flag("zipkin.web.query.limit", 10, "Default query limit for trace results")
+
   def newQueryClient(): ZipkinQuery.FutureIface =
     Thrift.newIface[ZipkinQuery.FutureIface]("ZipkinQuery=" + queryDest())
 
   def newJsonGenerator = new ZipkinJson
   def newMustacheGenerator = new ZipkinMustache(webResourcesRoot(), webCacheResources())
-  def newHandlers = new Handlers(newJsonGenerator, newMustacheGenerator)
+  def newQueryExtractor = new QueryExtractor(queryLimit.apply())
```
Comment (mosesn) [inline zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala]: maybe say `queryLimit()` instead of `queryLimit.apply()`?

Comment (codefromthecrypt) [inline zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala]: > maybe just say queryLimit() instead of queryLimit.apply()?
> 
> thx!

**Comment by mosesn** [comment]
lgtm, :shipit:

File: zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala (hunk 1 above)
Comment (codefromthecrypt) [inline zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala]: done

**Comment by codefromthecrypt** [comment]
thx for the review!

**Comment by rtyler** [comment]
_[shrug](https://www.youtube.com/watch?v=ussCHoQttyQ)_

**Comment by codefromthecrypt** [comment]
confirmed this works by deploying to a less used zone in prod at twitter

**Comment by codefromthecrypt** [comment]
oh yeah.. the syntax I verified is this `-zipkin.web.query.limit=100`

**Comment by eirslett** [comment]
One thing that could potentially cause some confusion is that we have different ways of configuring collector/query (with .Scala files) than web (flags). Would it be worth the effort to rewrite to one "universal" method?

**Comment by codefromthecrypt** [comment]
> One thing that could potentially cause some confusion is that we have
> different ways of configuring collector/query (with .Scala files) than web
> (flags). Would it be worth the effort to rewrite to one "universal" method?
> I noticed the inconsistency too. Refactoring how we do config in general is
> probably due as things have improved in the last few years. Care to open a
> top-level issue on how you think config ought be handled?

**Pull request merged**